src/components/view/NotificationsView/NotificationsView.tsx

```diff
@@ -13,7 +13,11 @@ export interface NotificationsViewProps {
 
 export type ViewPresentationType = "Modal" | "Push";
 
-export default function (props: NotificationsViewProps) {
+/**
+ * An out of the box view for notifications.
+ * Can be configured to show all, or isolated to email, push, or sms. 
```

Comment:
I think this component can only be configured to show a single notification type.  It just defaults to `Push` if you don't specify one.

Also, there is an extra `*` on the line below this.